### PR TITLE
[KYUUBI #2086] Add issue template for subtasks

### PR DESCRIPTION
**Pull request by Nick-0723**
### _Why are the changes needed?_
 Add issue template for subtasks

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/subtask.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Parent issue
+      placeholder: >
+        Please put the parent issue link here.
+    validations:
+      required: true
+
```
Comment (yaooqinn) [inline .github/ISSUE_TEMPLATE/subtask.yml:58]: we need a textarea for description of the subtask

Comment (Nick-0723) [reply .github/ISSUE_TEMPLATE/subtask.yml:66]: done, plz review

**Review by yaooqinn — APPROVED** [review]
LGTM, thanks

**Comment by yaooqinn** [comment]
thanks, merged to master

**Comment by yaooqinn** [comment]
https://github.com/apache/incubator-kyuubi/blob/master/.github/ISSUE_TEMPLATE/subtask.yml there are syntax errors, can you make a follow-up to fix them?

**Comment by Nick-0723** [comment]
> https://github.com/apache/incubator-kyuubi/blob/master/.github/ISSUE_TEMPLATE/subtask.yml there are syntax errors, can you make a follow-up to fix them?

oops, I get it

**Pull request closed without merging**